http/action/src/main/java/io/knotx/databridge/http/action/HttpAction.java

```diff
@@ -250,6 +262,7 @@ private void updateResponseMetadata(EndpointResponse response, ActionPayload pay
 
   private ActionPayload handleErrorResponse(ActionRequest request, String statusCode,
       String statusMessage) {
+    logOnError(request, statusCode, statusMessage);
```

Comment:
It happens when a service responds with status code different than HTTP 200. It means that we do not log:
- timeout exceptions when the HTTP client terminates the request
- invalid response data syntax exception (see DecodeException)

Please check `io.reactivex.Single#subscribe` in `io.knotx.databridge.http.action.HttpAction#apply`.